lib/presentation/ui/occurrences_time_widget.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class OccurrencesTimeWidget extends StatelessWidget {
+
+  final DateTime _startTime;
+  final DateTime _endTime;
+  final DateTime _lastUpdated;
+
+  const OccurrencesTimeWidget(this._startTime, this._endTime, this._lastUpdated);
+
+  static const String EMPTY_DATE_STRING = "--";
+  static const String FORMAT_DATE_DAY = "day";
+  static const String FORMAT_DATE_HOUR = "hour";
+  static const String FORMAT_DATE_FULL = "full";
+
+  String getFormattedDate(DateTime date, String typeOfFormat) {
+    String formattedDate = EMPTY_DATE_STRING;
+    DateFormat formatter;
+
+    if (date != null) {
+      switch(typeOfFormat) {
+        case 'full': {
+          formatter = new DateFormat('HH:mm yyyy-MM-dd');
+          break;
+        }
+        case 'day': {
+          formatter = new DateFormat('yyyy-MM-dd');
+          break;
+        }
+        case 'hour': {
+          formatter = new DateFormat('HH:mm');
+          break;
+        }
+        default: {
+
+        }
+      }
+
+      formattedDate = formatter.format(date);
+    }
+
+    return formattedDate;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
```

Comment:
Why do you need the material ancestor?

Reply:
I read and observed that wrapping the widgets with the Material widgets causes the yellow lines that appear on the Text widgets to disappear.

Without the Material ancestor:
![app](https://user-images.githubusercontent.com/23402988/67149809-6a219200-f2b8-11e9-8efe-0d17a8138a46.jpg)

If you want, I can commit the file with it removed.